.github/workflows/main.yml

```
@@ -45,14 +45,18 @@ jobs:
       - uses: actions/setup-dotnet@v4
         with:
           global-json-file: packages/nimble-blazor/global.json
-      - run: dotnet --info
-        working-directory: packages/nimble-blazor/
+          cache: true
+          cache-dependency-path: '**/packages.lock.json'
       - uses: actions/setup-node@v4
         with:
           node-version: '20'
           registry-url: 'https://registry.npmjs.org'
           cache: 'npm'
       - run: npm ci
+      - run: dotnet restore --locked-mode
+        working-directory: packages/nimble-blazor/
+      - run: dotnet --info
+        working-directory: packages/nimble-blazor/
```

Is this desired or leftover from debugging?

If it's useful to keep around lets just move it down so it's not groupeed under the `Install dependencies` comment

```suggestion

      # Print resolved dotnet info
      - run: dotnet --info
        working-directory: packages/nimble-blazor/
```

---

Well, it was intentionally introduced as part of [this PR](https://github.com/ni/nimble/pull/1283), with the reasoning that it should make debugging issues related to versioning easier.